packages/0xcert-ethereum-bitski-frontend-provider/package.json

```diff
@@ -0,0 +1,84 @@
+{
+  "name": "@0xcert/ethereum-bitski-frontend-provider",
+  "version": "1.0.1-alpha1",
+  "description": "Implementation of Bitski frontend communication provider for the Ethereum blockchain.",
```

Comment:
`"description": "Implementation of Bitski front-end communication provider for the Ethereum blockchain.",`